github-auth-hook.js

```diff
@@ -14,22 +15,71 @@ passport.use(
         },
 
         (accessToken, refreshToken, profile, done) => {
-            console.log("Access token: " + accessToken)
             if (!profile.emails) {
                 // user can choose to not display any email, then use a default one as unleash required it
                 profile.emails.push(`${displayName}@unknown.com`);
             }
-            done(
-                null,
-                new User({
-                    name: profile.displayName,
-                    email: profile.emails[0].value,
-                })
+
+            let githubOrg = process.env.GITHUB_ORG ? process.env.GITHUB_ORG : 'rhdt-toggles-test';
+            let githubOrgTeam = process.env.GITHUB_ORG_TEAM ? process.env.GITHUB_ORG : 'toggles-admin-test';
```

Comment:
do you mean:
let githubOrgTeam = process.env.GITHUB_ORG_TEAM ? process.env.GITHUB_ORG_TEAM : 'toggles-admin-test';

Reply:
ah right :/ let me fix that!